dev/load-test/test-recipes/csn-load-test/cost_graph.py

```
@@ -0,0 +1,153 @@
+        return
+
+    def series(rows):
+        t0 = rows[0][0]
+        return ([(ts - t0) / 60.0 for ts, _, _ in rows],
+                [tot - sus for _, tot, sus in rows],
+                [sus for _, _, sus in rows])
+
+    fig, ax = plt.subplots(figsize=(11, 5.5))
+    m, run, sus = series(csn_rows)
+    ax.fill_between(m, 0, run, color="#d9534f", alpha=0.7, label="CSN running")
+    ax.fill_between(m, run, [r+s for r, s in zip(run, sus)], color="#5bc0de",
```

Ruff (0.15.18)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)